code/src/components/AddToDoInput.js

```diff
@@ -0,0 +1,68 @@
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
```

Comment:
I think the "import { useDispatch } from 'react-redux';" and the "import { todolist } from 'reducers/todolist';" dependencies should have a blank space between each other. I think it works as with components and that reducers/todolist is a local dependency. 📱

Please correct me if I'm wrong! 😁